example/app/config.yml

```diff
@@ -618,6 +618,7 @@ gitc:
 mvd:
   prefix: mvd
   prefixNoDash: mvd
+  useWorkflowLambdaVersions: false
```

Comment:
Does this need to be a universal config change, or was it something specific to your stack?

Reply:
Specific to my stack in that it made testing updates to the CMA layer simpler.